src/Fabulous/Core.fs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace Fabulous
```

Comment:
You mentioned about renaming this file. Maybe `Primitives` could be good?

Reply:
I like it! Let's do `Primitives.fs`